### DISCUSSION TITLE
New v0.1.0 version

### DISCUSSION THREAD

**Discussion by 4rtzel** [4rtzel/poe-archnemesis-scanner#31] [Announcements]
* Update images for Entangler, Innocence-Touched, Kitava-Touched, Lunaris-Touched, Opulent, Solaris-Touched to match their in-game equivalents.
* Add copy recipe to clipboard feature (thanks https://github.com/alexberryman).
* Fix an issue where highlights would stay on the screen when interacting with the recipe tree.
* Add a way to hide the tree by clicking on the top recipe.
* Add a list of recipes that the currently selected recipe is used in.
* Add an error message box pop up when start the tool without game running.
* Add a warning message box pop up when the tool wasn't able to detect game resolution.
* Add "Other languages" section to the README.

<hr /><em>This discussion was created from the release <a href='https://github.com/4rtzel/poe-archnemesis-scanner/releases/tag/v0.1.0'>v0.1.0: Display message box when the game is not running. Display message box…</a>.</em>

**Comment by Karl-Ehlhardt** [comment]
Man, i've been trying to get this to work in c# by matching pixels, how did you end up getting the information from the screen?

**Reply by 4rtzel** [reply]
The tool simple grabs the screenshot of the window and trying to match the source images on it. I'm using OpenCV for image searching and that alleviate a lot of work.

**Reply by Noctis-** [reply]
OpenCV to the rescue ... i've toyed with the idea of doing the same in C# and then saw you're tool ... It rocks :)
Anyhows, gratz on the new version and thanks for all the fixes and hard work ... i'll give it a spin later today :)
